src/addValidator.ts

```diff
@@ -63,7 +63,7 @@ export async function getUnsignedAddValidator(
   }
 }
 
-async function getAddValidatorParams(
+export async function getAddValidatorParams(

```

Comment:
since the function is export please add function description in comment as documentation .